wiki/Layout-component/Lead-space.md

```diff
@@ -13,6 +13,7 @@ The [[Lead space]] is a top-level component section that appears at the beginnin
 
 - [[Button group]]
 - [[Tag group]]
+- [[Carbon breadcrumb]]
```

Comment:
@shixiedesign with lead space wrapper I think we could explore removing these dependencies in lead space and lead space search.

Reply:
Following `background media` component discussion, we should leave the breadcrumb in here for a bit longer.

Reply:
no problem I'll get a change in for that. thanks!